### ISSUE TITLE
Last publish 2.3.2 broken

### ISSUE THREAD

**Issue by p0psicles** [dzcpy/transliteration#237]
Transliteration is used by vuepress-bar as a dependency.
I'm when running the vuepress-bar plugin in my vuepress site, i'm getting the following error:

```js
Exception has occurred: Error: Cannot find module 'tslib'
Require stack:
- //t3x/accordion/node_modules/transliteration/dist/node/src/node/index.js
- //t3x/accordion/node_modules/vuepress-bar/lib/index.js
- //t3x/accordion/docs/typo3/extension/accordion/.vuepress/config.js
- //t3x/accordion/node_modules/@vuepress/core/lib/node/loadConfig.js
```

This hasn't been an issue with the prior verison.

**Comment by dzcpy** [comment]
Ah, sorry for the error, I'll fix it asap

**Comment by dzcpy** [comment]
Fixed in 2.3.3
